packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -213,25 +213,26 @@ $search-transition-timing: 95ms;
   }
 
   .#{$prefix}--masthead--sticky.#{$prefix}--masthead--sticky__l1 {
+    top: -98px;
+
     @include carbon--breakpoint-up(800px) {
       top: -48px;
     }
   }
 
-  .#{$prefix}--masthead--sticky__l1 + .#{$prefix}--dotcom-shell {
-    @include carbon--breakpoint-up(800px) {
-      .#{$prefix}--tableofcontents__sidebar {
-        top: 98px;
-      }
-    }
+  .#{$prefix}--masthead--sticky__l1
+    + .#{$prefix}--dotcom-shell
+    .#{$prefix}--tableofcontents__sidebar {
+    top: 98px;
```

Comment:
Might be better using the `rem()` function in the stylesheet to keep things consistent across styles?

Reply:
@IgnacioBecerra should I convert the existing styles there to `rem` as well?